single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -632,7 +632,9 @@ private void createParametersPane() {
                 pane.getDiagramView().setZoom(1.0); // 100%
             }
         });
-        FlowPane buttonsPane = new FlowPane(fitToContent, resetZoom);
+        GridPane buttonsPane = new GridPane();
+        buttonsPane.add(fitToContent, 0, 0);
+        buttonsPane.add(resetZoom, 1, 0);
```

Comment:
Using GridPane instead of FlowPane make appears horizontal scrollbar only if needed